### PR TITLE
Improved Player UI (custom player skin)

### PR DESCRIPTION
**Pull request by DispatchCommit**
Work In Progress - Feedback / Questions / Suggestions Welcomed

Testing Needed on iOS devices and Mac Safari Browser to verify no breaking styling occur as a result of CSS conflicts.

UI Preview:

image

### CODE REVIEW

File: ui/scss/component/_button.scss [hunk 1]
```diff
@@ -104,20 +104,17 @@
   @media (min-width: $breakpoint-medium) {
     display: block;
     order: 1;
-    width: 1.25rem;
-    height: 1.25rem;
-    margin-top: 0.3rem;
-    margin-left: 0.3rem;
```
Comment (seanyesmunt) [inline ui/scss/component/_button.scss:110]: 👍

File: ui/scss/component/_button.scss [hunk 2]
```diff
@@ -104,20 +104,17 @@
...
-    // Need to repeat these styles because of videojs weirdness
-    background: black
-      url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='18' height='14' viewBox='0 -2 24 24' fill='none' stroke='white' stroke-width='2' stroke-linecap='round' stroke-linejoin='round' class='feather feather-monitor'%3E%3Crect x='2' y='3' width='20' height='14' rx='2' ry='2'%3E%3C/rect%3E%3Cline x1='8' y1='21' x2='16' y2='21'%3E%3C/line%3E%3Cline x1='12' y1='17' x2='12' y2='21'%3E%3C/line%3E%3C/svg%3E")
-      no-repeat center;
+    // Need to repeat these styles because of video.js weirdness
+    // TODO: verify that this "weirdness" actually exists
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='18' height='14' viewBox='0 -2 24 24' fill='none' stroke='white' stroke-width='2' stroke-linecap='round' stroke-linejoin='round' class='feather feather-monitor'%3E%3Crect x='2' y='3' width='20' height='14' rx='2' ry='2'%3E%3C/rect%3E%3Cline x1='8' y1='21' x2='16' y2='21'%3E%3C/line%3E%3Cline x1='12' y1='17' x2='12' y2='21'%3E%3C/line%3E%3C/svg%3E");
```
Comment (seanyesmunt) [inline ui/scss/component/_button.scss:117]: Without these rules, the icon would disappear (turn transparent I think) when the button had focus.

File: ui/component/viewers/videoViewer/internal/plugins/videojs-overlay/plugin.scss [hunk 3]
```diff
@@ -81,5 +81,163 @@
   .vjs-no-flex .vjs-overlay-right {
     margin-top: $offset-v;
   }
+}*/
+
+$accent-color: var(--color-primary);
```
Comment (seanyesmunt) [inline ui/component/viewers/videoViewer/internal/plugins/videojs-overlay/plugin.scss:86]: Please just use `var(--color-primary)` everywhere, or add a new line to the light/dark.scss called something like `--video-accent-color` (even if that is still `--color-primary` under the hood

File: ui/scss/component/_file-render.scss [hunk 4]
```diff
@@ -387,7 +387,7 @@
   }
 
   .vjs-control-bar {
-    background-color: rgba(0, 0, 0, 0.8);
+    // background-color: rgba(0, 0, 0, 0.8);
```
Comment (seanyesmunt) [inline ui/scss/component/_file-render.scss:390]: extra comment

**Review by seanyesmunt — APPROVED** [review]
This looks good. Tested on Safari and works fine. Please squash/rebase your branch to get rid of all the merge commits.

**Comment by infinite-persistence** [comment]
image

The color for the volume slider seems inverted (highlight color used as the "background")

**Pull request merged**